frontend/src/app/header.tsx

```diff
@@ -17,14 +18,17 @@ export default function ConnectButton() {
 
 const VoucherNotification: FC = () => {
     return (
-        <ActionIcon
-            variant="default"
-            aria-label="Settings"
-            component="a"
-            href="/voucher"
-        >
-            <FcMoneyTransfer size="24px" />
-        </ActionIcon>
+        <Tooltip label="List vouchers">
+            <ActionIcon
+                variant="filled"
+                aria-label="Settings"
```

Comment:
`aria-label` is an accessibility feature that should describe an interactive element. I don't think the word "settings" correctly labels the "List vouchers" icon.

Reply:
Good catch, @guidanoli ! Damned copy and paste, in this case.
I removed the attrib since we are not committed to adapt the whole system to screen readers, ok?

Reply:
Fine with me!